tests/cases/classes/property-type

```diff
@@ -0,0 +1,39 @@
+---DESCRIPTION---
+Property types must be declared
```

Comment:
I haven't done this in the past, but should help static analysis (and formal types should be in PHP 7.4).